README.md

```diff
@@ -15,20 +15,56 @@
 |:----------------:|:----------------------------:|
 | v0.1.*           | v1.5.*                       |
 
-## Installation
+## Using this starter kit
 
-Add the following dependency to your Akeneo PIM project with this shell command:
+### Context
+To write an extension, you will need to integrate your extension in an existing PIM installation.
```

Comment:
Your extension is meant to work as part of an existing PIM installation, thus we will use PIM CE to host the connector.